add stadtwald layer for Meldemichel clerks (#213)
Dataport/polar: packages/clients/meldemichel/CHANGELOG.md

## unpublished

> - Feature: Added `stadtwaldActive` as startup parameter for `createMap` object and `meldemichel/setMapState` action. Search API.md for `stadtwaldActive` regarding details.

Reviewer: ```suggestion
- Feature: Add `stadtwaldActive` as startup parameter for `createMap` object and `meldemichel/setMapState` action. Refer to the API.md regarding further details.
```
